go/nogo.rst

```diff
@@ -71,15 +71,19 @@ want to run.
         visibility = ["//visibility:public"],
     )
 
-Pass a label for your `nogo`_ target to ``go_register_toolchains`` in your
+Pass a label for your `nogo`_ target to ``go_register_nogo`` in your
 ``WORKSPACE`` file. When using ``MODULE.bazel``, see the Bzlmod_ documentation
 instead.
 
 .. code:: bzl
 
-    load("@io_bazel_rules_go//go:deps.bzl", "go_rules_dependencies", "go_register_toolchains")
+    load("@io_bazel_rules_go//go:deps.bzl", "go_rules_dependencies", "go_register_nogo")
     go_rules_dependencies()
-    go_register_toolchains(nogo = "@//:my_nogo") # my_nogo is in the top-level BUILD file of this workspace
+    go_register_nogo(
```

Comment:
How does this interact with `go_register_toolchain` calls that users make to e.g. select a different Go version? Could you document an example?

Reply:
Added an example. Previously you would have two calls to `go_register_toolchains()` and now you have one call to `go_register_toolchains` and one to `go_register_nogo`